content/docs/dvclive/user-guide/dvclive-with-dvc.md

```diff
@@ -0,0 +1,129 @@
+# DVCLive with DVC
+
+Even though DVCLive does not require DVC, they can integrate in several useful
+ways:
+
+- The [_outputs_](#outputs) DVCLive produces can be fed as
+  `dvc plots`/`dvc metrics`, making it easier to add metrics logging to DVC
+  <abbr>stages</abbr>. Those same outputs can be visualized in
+  [_DVC Studio_](#dvc-studio)
+- You can monitor model performance in realtime with the
+  [_HTML report_](#html-report) that DVCLive generates when used alongside DVC.
+- DVCLive is also capable of generating [_checkpoint_](#checkpoints) signal
+  files used by DVC <abbr>experiments<abbr>.
+
+## Setup
+
+We will refer to a training script (`train.py`) already using `dvclive`:
+
+```python
+# train.py
+
+import dvclive
+
+for epoch in range(NUM_EPOCHS):
+    train_model(...)
+    metrics = evaluate_model(...)
+
+    for metric_name, value in metrics.items():
+        dvclive.log(metric_name, value)
+
+    dvclive.next_step()
+```
+
+> 💡 When using DVCLive in a DVC project, there is no need for manual
+> initialization of DVCLive (no [`dvclive.init()`] call). DVCLive will use the
+> settings provided by DVC.
+
+Let's use `dvc stage add` to create a stage to wrap this code (don't forget to
+`dvc init` first):
+
+```dvc
+$ dvc stage add -n train --live training_metrics
+                -d train.py python train.py
+```
+
+`dvc.yaml` will contain a new `train` stage with the [`DVCLive configuration`]
+(in the `live` field):
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    live:
+      training_metrics:
+        summary: true
+        html: true
+```
+
+The value passed to `--live` (`training_metrics`) became the directory `path`
+for DVCLive to write logs in, and DVC will now
+[track](/doc/use-cases/versioning-data-and-model-files) it. Other supported
+command options for the DVC integration:
+
+- `--live-no-cache <path>` - specify a DVCLive log directory `path` but don't
+  tracked it with DVC. Useful if you prefer to track it with Git.
+- `--live-no-summary` - passes `summary=False` to [`DVCLive configuration`].
+- `--live-no-html` - passes `html=False` to [`DVCLive configuration`].
+
+> Note that these are convenience CLI options. You can still use
+> [`dvclive.init()`] manually, which will override any options sent to
+> `dvc stage add`. Just be careful to match the `--live` value (CLI) and `path`
+> argument (code). Also, note that summary files are never tracked by DVC
+> automatically.
+
+Run the training with `dvc repro`:
+
+```bash
+$ dvc repro train
+```
+
+## Outputs
+
+After that's finished, you should see the following content in the project:
+
+```bash
+$ ls
+dvc.lock  training_metrics       training_metrics.json
+dvc.yaml  training_metrics.html  train.py
+```
+
+The `.tsv` files generated under `training_metrics` can be visualized with
+`dvc plots`.
+
+In addition, `training_metrics.json` can be used by `dvc metrics`.
+
+### DVC Studio
```

Comment:
Does this actually require DVC integration? Might be a question for the studio team.

Reply:
It is not really required. 

Would it make sense to have a separated `dvclive` with `DVC Studio`?? I'm not sure about the narrative we  should have here.

Reply:
It looks like metrics can be read without dvc but not the plots. I think let's leave it as is for now and we can revisit later.